recipes/make/all/conanfile.py

```diff
@@ -38,8 +39,19 @@ def package_id(self):
         del self.info.settings.compiler
 
     def source(self):
-        get(self, **self.conan_data["sources"][self.version],
-            destination=self.source_folder, strip_root=True)
+        for data in self.conan_data["sources"][self.version]:
+            url, sha = data.values()
+            filename = url[url.rfind("/")+1:]
+            try:
+                download(self, url, filename, sha256=sha, retry=0)
+            except ConanException as e:
+                if "NewConnectionError" in str(e):
```

Comment:
I don't know how / if you can filter exceptions in python so I used this. Better than nothing, lmk if there's a better way.

Reply:
There's no need for this. The `get`/`download` functions should already be able to handle mirrors

Reply:
Or does this come from the current get implementation not handling this failing mode properly?

Reply:
@RubenRBS Could you point to a recent example? All my research seemed to indicate mirrors coming from conandata.yml needed some custom logic in the recipe, but a lot of the info was old (google is bad nowadayts). If it doesn't that's great :)

Reply:
Roger :] I'll ping you re-review when I have a some time to teak